bbc/simorgh
# Navigaton bar updates for Nepali and Vietnamese

Resolves JIRA n/a

Updated BBC Vietnamese navigation bar to remove no longer needed link to FIX. Edited the Radio programmes link on the Nepali nav bar to avoid the unnecessary extra click taking users directly to the radio programme page.

Code changes
======

- removed Nhịp sống mới link in the navigation section of src/app/lib/config/services/vietnamese.ts
- replaced the link to /nepali/media-54029171 for/nepali/bbc_nepali_radio/programmes/p0340xzv  on the navigation section of src/app/lib/config/services/nepali.ts
- updated snapshots

Testing
======
1. Open the BBC Vietnamese front page, the Nhịp sống mới (/vietnamese/magazine-54029179) link should not be shown between Kinh tế and Thể thao items
2. Open the BBC Nepali front page, the second link should take you to /nepali/bbc_nepali_radio/programmes/p0340xzv 


Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
